service/src/main/java/com/fecklessweasel/service/datatier/UniversityTable.java

```diff
@@ -85,4 +87,22 @@ public static ResultSet lookupUniversity(Connection connection, int uid)
             throw new ServiceException(ServiceStatus.DATABASE_ERROR, ex);
         }
     }
+
+    /**Looks up the University in the database bassed off of the id**/
+    public static ResultSet lookupUniversityID(Connection connection, int univid)
+            throws ServiceException{
+
+        CodeContract.assertNotNull(univid, "univid");
```

Comment:
There is no point to this line, univid cannot be null. It is an int. However, there should be an assertNotNull for connection which is a reference type.